From 6f1e7424a2b435a5f4c33277dd745c5dd917a694 Mon Sep 17 00:00:00 2001
From: Tri Luong <tri.luong.te@rvc.renesas.com>
Date: Wed, 8 Aug 2018 08:38:53 +0700
Subject: [PATCH 600/628] pfc-r8a7742: Add missing AVB_TX_ER pin function.

Fix wrong definition SCIFA2_RXD_B and VI1_CLK_B pin function.

Signed-off-by: Tri Luong <tri.luong.te@rvc.renesas.com>
---
 drivers/pinctrl/sh-pfc/pfc-r8a7742.c | 14 +++++++-------
 1 file changed, 7 insertions(+), 7 deletions(-)

diff --git a/drivers/pinctrl/sh-pfc/pfc-r8a7742.c b/drivers/pinctrl/sh-pfc/pfc-r8a7742.c
index 6e93a18..ec605c1 100644
--- a/drivers/pinctrl/sh-pfc/pfc-r8a7742.c
+++ b/drivers/pinctrl/sh-pfc/pfc-r8a7742.c
@@ -1511,8 +1511,8 @@ static const unsigned int avb_mii_pins[] = {
 	RCAR_GP_PIN(2, 2),
 
 	RCAR_GP_PIN(2, 7), RCAR_GP_PIN(2, 8), RCAR_GP_PIN(2, 9),
-	RCAR_GP_PIN(2, 10), RCAR_GP_PIN(3, 8), RCAR_GP_PIN(3, 10),
-	RCAR_GP_PIN(3, 12),
+	RCAR_GP_PIN(2, 10), RCAR_GP_PIN(3, 8), RCAR_GP_PIN(3, 9),
+	RCAR_GP_PIN(3, 10), RCAR_GP_PIN(3, 12),
 };
 static const unsigned int avb_mii_mux[] = {
 	AVB_TXD0_MARK, AVB_TXD1_MARK, AVB_TXD2_MARK,
@@ -1522,8 +1522,8 @@ static const unsigned int avb_mii_mux[] = {
 	AVB_RXD3_MARK,
 
 	AVB_RX_ER_MARK, AVB_RX_CLK_MARK, AVB_RX_DV_MARK,
-	AVB_CRS_MARK, AVB_TX_EN_MARK, AVB_TX_CLK_MARK,
-	AVB_COL_MARK,
+	AVB_CRS_MARK, AVB_TX_EN_MARK, AVB_TX_ER_MARK,
+	AVB_TX_CLK_MARK, AVB_COL_MARK,
 };
 static const unsigned int avb_gmii_pins[] = {
 	RCAR_GP_PIN(0, 8), RCAR_GP_PIN(0, 9), RCAR_GP_PIN(0, 10),
@@ -2681,7 +2681,7 @@ static const unsigned int scifa2_ctrl_mux[] = {
 };
 static const unsigned int scifa2_data_b_pins[] = {
 	/* RXD, TXD */
-	RCAR_GP_PIN(5, 13), RCAR_GP_PIN(5, 16),
+	RCAR_GP_PIN(5, 17), RCAR_GP_PIN(5, 16),
 };
 static const unsigned int scifa2_data_b_mux[] = {
 	SCIFA2_RXD_B_MARK, SCIFA2_TXD_B_MARK,
@@ -3514,10 +3514,10 @@ static const unsigned int vin1_clkenb_mux[] = {
 	VI1_CLKENB_MARK,
 };
 static const unsigned int vin1_clk_pins[] = {
-	RCAR_GP_PIN(3, 15),
+	RCAR_GP_PIN(2, 9),
 };
 static const unsigned int vin1_clk_mux[] = {
-	VI1_CLK_B_MARK,
+	VI1_CLK_MARK,
 };
 /* - VIN2 ----------------------------------------------------------------- */
 static const union vin_data vin2_data_pins = {
-- 
2.7.4

